tracing-subscriber/src/field/delimited.rs

```diff
@@ -0,0 +1,170 @@
+use super::{MakeVisitor, VisitFmt, VisitOutput};
+
+use std::fmt;
+use tracing_core::field::{Field, Visit};
+
+#[derive(Debug, Clone)]
+pub struct Delimited<D, V> {
+    delimiter: D,
+    inner: V,
+}
+
+#[derive(Debug)]
+pub struct VisitDelimited<D, V> {
```

Comment:
So if I am understanding this correctly, this would mean that you'd simply have to instantiate a `VisitDelimited` for a new visitor, without having to wrap the Visitor construction in a separate type?